src/Data/Nat/Divisibility/Core.agda

```diff
@@ -14,11 +14,14 @@ module Data.Nat.Divisibility.Core where
 
 open import Data.Nat.Base using (ℕ; _*_; _<_; NonTrivial)
 open import Data.Nat.Properties
-open import Level using (0ℓ)
 open import Relation.Nullary.Negation using (¬_)
 open import Relation.Binary.Core using (Rel)
-open import Relation.Binary.PropositionalEquality
-  using (_≡_; refl; sym; cong₂; module ≡-Reasoning)
+open import Relation.Binary.PropositionalEquality.Core
+  using (_≡_; refl)
```

Comment:
I suppose that this might even be simplified further in favour of the `Builtin` version...?

Reply:
I think it better we try and avoid depending on `Builtin` modules unless we absolutely have to. Otherwise, we become more vulnerable to changes to the Agda API.